kvbc/include/ReplicaImp.h

```diff
@@ -176,6 +176,7 @@ class ReplicaImp : public IReplica,
   // secretsManager_ can be nullptr. This means that encrypted configuration is not enabled
   // and there is no instance of SecretsManagerEnc available
   const std::shared_ptr<concord::secretsmanager::ISecretsManagerImpl> secretsManager_;
+  std::unique_ptr<concord::kvbc::StReconfigurationHandler> stReconfigurationSM_;
```

Comment:
Is it in use yet?

Reply:
currently all the implementation is empty (but the callback is being called when ST is done)
This object exists only for the state transfer callback